packages/react/src/use-chat.ts

```diff
@@ -52,6 +52,12 @@ export type UseChatHelpers = {
    * Abort the current request immediately, keep the generated tokens if any.
    */
   stop: () => void;
+
+  /**
+   * Resume an ongoing chat generation stream. This does not resume an aborted generation.
+   */
+  experimental_resume: () => Promise<string | null | undefined>;
```

Comment:
What is the meaning of the promise return values?

Reply:
I made it return void